frontend/src/components/UserlocationCard.tsx

```diff
@@ -0,0 +1,58 @@
+import {Userlocation} from "./Entities.ts";
+
+type UserlocationCardProps = {
+    userlocation: Userlocation
```

Comment:
```suggestion
    readonly userlocation: Userlocation
```
dann ist dein code small an der Stelle weg und da du den Wert hier nur anzeigst, wäre es auch gut.